### ISSUE TITLE
[v5] PSQLException: ERROR after migration

### ISSUE THREAD

**Issue by duzztie** [reportportal/reportportal#920]
After migrating reportportal from 5.0 to 5.1 there are errors in service-api logs(earlier reportportal was migrated to 5.0 from 4.3):
```
2020-04-08 14:47:46.049 ERROR 9 --- [nio-8585-exec-4] c.e.t.r.c.e.rest.DefaultErrorResolver    : could not extract ResultSet
org.hibernate.exception.SQLGrammarException: could not extract ResultSet
	at org.hibernate.exception.internal.SQLStateConversionDelegate.convert(SQLStateConversionDelegate.java:103)
	at org.hibernate.exception.internal.StandardSQLExceptionConverter.convert(StandardSQLExceptionConverter.java:42)
	at org.hibernate.engine.jdbc.spi.SqlExceptionHelper.convert(SqlExceptionHelper.java:113)
```
...
```
Caused by: org.postgresql.util.PSQLException: ERROR: column project0_.allocated_storage does not exist
  Position: 35
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2578)
	at org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:2313)
	at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:331)
	at org.postgresql.jdbc.PgStatement.executeInternal(PgStatement.java:448)
	at org.postgresql.jdbc.PgStatement.execute(PgStatement.java:369)
	at org.postgresql.jdbc.PgPreparedStatement.executeWithFlags(PgPreparedStatement.java:159)
	at org.postgresql.jdbc.PgPreparedStatement.executeQuery(PgPreparedStatement.java:109)
	at com.zaxxer.hikari.pool.ProxyPreparedStatement.executeQuery(ProxyPreparedStatement.java:52)
	at com.zaxxer.hikari.pool.HikariProxyPreparedStatement.executeQuery(HikariProxyPreparedStatement.java)
	at org.hibernate.engine.jdbc.internal.ResultSetReturnImpl.extract(ResultSetReturnImpl.java:57)
	... 155 common frames omitted
```

**Comment by pbortnik** [comment]
Have you used a default docker-compose.yml file?
Do you have a stopped container 'db-scripts'? To check that use the command 'docker ps -a'
If yes, share with us what you have in the container's log.

**Comment by duzztie** [comment]
Thank you for your reply. Reportportal is in k8s, for running db-scripts when migrating 4.3->5.0.0 I copied sections "db-scripts" and "postgres" into separate docker-compose.yml from here https://github.com/reportportal/reportportal/blob/master/docker-compose.yml. I used tag 5.0.0 for image: reportportal/migrations. Here are the container's logs:

$ docker logs c99c3b20ba7b
timeout: unrecognized option: t
BusyBox v1.31.1 () multi-call binary.

Usage: timeout [-s SIG] SECS PROG ARGS

Runs PROG. Sends SIG to it if it is not gone in SECS seconds.
Default SIG: TERM.
wait-for-it.sh: timeout occurred after waiting 15 seconds for 172.17.224.172:5432
0/u extensions (54.741866ms)
1/u initialize_schema (362.505898ms)
2/u initialize_quartz_schema (420.173886ms)
3/u default_data (441.944747ms)
4/u size_limitations (461.072314ms)
5/u test_case_id_type (489.710536ms)
6/u retries_handling (505.407208ms)
